reactive/webserver/webserver/src/main/java/io/helidon/reactive/webserver/SocketConfiguration.java

```diff
@@ -542,6 +570,11 @@ public SocketConfiguration build() {
                 throw new ConfigException("Socket name must be configured for each socket");
             }
 
+            if (null == requestedUriDiscoveryContextBuilder) {
+                requestedUriDiscoveryContextBuilder = RequestedUriDiscoveryContext.builder()
+                        .socketId(WebServer.DEFAULT_SOCKET_NAME);
```

Comment:
Is the context same for all sockets? I would expect something like: 
```
.socketId(UNCONFIGURED_NAME.equals(name) ? WebServer.DEFAULT_SOCKET_NAME : name);
```

Reply:
fixed